Python_sem_4/DZ1.py

```diff
@@ -0,0 +1,17 @@
+# Напишите функцию для транспонирования матрицы
+import numpy as np
+matrix = [[1, 2], [31, 4], [17, 15], [20, 48]]
+
+def my_func (mat):
+    zip_matrix = zip(*mat)
+    transpon_matrix = [list(row) for row in zip_matrix]
+    return transpon_matrix
```

Comment:
Вы абсолютно верно решили использовать функцию `zip`. Но решение можно уместить в одну строку.
```suggestion
    return [*map(list, zip(*mat))]
```